server/website/website/tasks/async_tasks.py

```diff
@@ -138,7 +138,11 @@ def clean_knob_data(knob_matrix, knob_labels, session):
         for knob in missing_columns:
             knob_object = KnobCatalog.objects.get(dbms=session.dbms, name=knob, tunable=True)
             index = knob_cat.index(knob)
-            matrix = np.insert(matrix, index, knob_object.default, axis=1)
+            try:
+                default_val = float(knob_object.default)
+            except ValueError:
+                default_val = 0
+            matrix = np.insert(matrix, index, default_val, axis=1)
```

Comment:
This is for handling the ValueError caused by converting enum default string to float.

Reply:
I think it's better to log a error message here if ValueError

Reply:
It is normal for enum & bool type knob to have none min/max. Yet I think we can log the missing column.